msa-persistence/src/main/java/cz/muni/fi/pa165/monsterslayeragency/dao/RequestDaoImpl.java

```diff
@@ -0,0 +1,77 @@
+package cz.muni.fi.pa165.monsterslayeragency.dao;
+
+import cz.muni.fi.pa165.monsterslayeragency.entities.Request;
+import cz.muni.fi.pa165.monsterslayeragency.entities.User;
+import org.springframework.stereotype.Repository;
+import org.springframework.transaction.annotation.Transactional;
+
+import javax.persistence.EntityManager;
+import javax.persistence.NoResultException;
+import javax.persistence.PersistenceContext;
+import java.util.List;
+
+/**
+ * @author Filip Daniel Fedin
+ */
+
+@Repository
+@Transactional
+public class RequestDaoImpl implements RequestDao {
+    @PersistenceContext
+    private EntityManager em;
+
+    @Override
+    public void addRequest(Request request) {
+        if (request != null) {
+            em.persist(request);
+            return;
+        }
+        throw new IllegalArgumentException("Request is null");
```

Comment:
I would prefer it the other way (same as you did in 'find' methods):
```
if (request == null) {
    throw ...
}

em.persist(request)
```

But it's up to you.